src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2190,6 +2191,15 @@ export class ResultsViewPageStore {
         return parsedQString.query.session_id;
     }
 
+    private getCytoband(hugoGeneSymbol:string):string {
```

Comment:
should be `readonly referenceGeneHugoSymbolToCytoband =  remoteData`. need to avoid these linear time scans in function calls when they can be precomputed once into a map and then constant time lookup every time afterwards.